configure.py

```diff
@@ -2389,7 +2386,7 @@ def create_building_system(args):
 
     ninja = find_ninja()
     with open(args.buildfile, 'w') as f:
-        scylla_product, scylla_version, scylla_release = generate_version(args.date_stamp)
+        scylla_product, scylla_version, scylla_release = generate_version()
```

Comment:
@tchaikov This is being used in Scylla-pkg when we building both `x86` and `aarch64` , we use it to verify that both package names, will be identical. 

See https://github.com/scylladb/scylla-pkg/blob/next/scripts/jenkins-pipelines/groovy_modules/build.groovy#L152

Reply:
This will break our Build process (and also gating once we merge https://github.com/scylladb/scylla-pkg/pull/3521

Reply:
@yaronkaikov hi Yaron, I knew it is used. And then it’s been broken for more than one year.

Reply:
I don't understand what is broken. See https://jenkins.scylladb.com/job/scylla-master/job/build/1677/consoleFull for example: 
We are running this command:
```
 Building Scylla with dist - command: bash -c 'set -o pipefail; /jenkins/workspace/scylla-master/build/scylla/tools/toolchain/dbuild    -- ./configure.py --date-stamp 20231027 --debuginfo 1 --tests-debuginfo 1 2>&1 | tee output-build-aarch64.txt'
```
And then we get 
```
 5.5.0~dev-0.20231027.227136ddf54f
```

So it is working, unless we are talking on 2 different things :-)